az-core/src/main/java/azkaban/Constants.java

```diff
@@ -144,6 +144,40 @@
   // AZ_HOME in containerized execution
   public static final String AZ_HOME = "AZ_HOME";
 
+  // Azkaban event reporter constants
+  public static final String FLOW_NAME = "flowName";
```

Comment:
Can you please add static class for all these event reporter constants? For e.g. EventReporterConstants something?

Reply:
created a static class named EventReporterConstants